PKHeX/Saves/SaveUtil.cs

```diff
@@ -544,7 +562,7 @@ public static string detectSaveFile()
                 possiblePaths.AddRange(getSavesFromFolder(Path.Combine(pathCache), false));
 
             // return newest save file path that is valid (oh man)
-            return possiblePaths.OrderByDescending(f => new FileInfo(f).LastWriteTime).FirstOrDefault(p => getVariantSAV(File.ReadAllBytes(p)).ChecksumsValid);
+            return possiblePaths.OrderByDescending(f => new FileInfo(f).LastWriteTime).FirstOrDefault(p => isVariantSAVValid(File.ReadAllBytes(p)));
```

Comment:
Should be:
`.FirstOrDefault(p => getVariantSAV(File.ReadAllBytes(p))?.ChecksumsValid ?? false);`
Removes the need for the extra method. Basically just null check the save in the linq instead of a separate method.